projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -102,10 +102,11 @@ export class EditorComponent extends AbstractCanDeactivateComponent implements O
   @Input()
   editorSettings = {
     longMode: false,  // editor long mode
+    getHeaders: null,      // optional headers to use when loading a model
```

Comment:
Wouldn't it be better to put a "Get" key with a substructure? That would allow us to add other configurations in the future without having to repeat the term "Get"
Perhaps:
```
{
  methods: {
   get: {
     headers: {
       Accept: 'application/json'
     }
   }
}
```

Reply:
An issue has been opened: https://github.com/rero/rero-ils/issues/3947